src/util.ts

```diff
@@ -73,7 +73,14 @@ const isAttributeArray = (value: any) => {
   if (!Array.isArray(value)) {
     return false;
   }
-  return value.filter((e) => e.key && e.value).length !== 0;
+
+  return (
+    value.filter(
+      (e) =>
+        Object.prototype.hasOwnProperty.call(e, "key") &&
+        Object.prototype.hasOwnProperty.call(e, "value")
```

Comment:
Instead of using `hasOwnProperty()` you can use [`Object.hasOwn()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn), which is intended as a modern replacement.
```suggestion
        Object.hasOwn(e, "key") &&
        Object.hasOwn(e, "value")
```

Reply:
Okay didn't know about that... let's do that